client/state/selectors/get-site-feed-url.ts

```diff
@@ -0,0 +1,15 @@
+import 'calypso/state/data-layer/wpcom/sites/scan';
+import getSiteUrl from './get-site-url';
+import type { AppState } from 'calypso/types';
+
+/**
```

Comment:
I am not much familiar with Jetpack Scan. What is the main purpose of this function? I mean it is clear to me what it does in its core but I don't see it used elsewhere in the code. 🤔